.github/workflows/rumble-cve-data.yaml

```diff
@@ -75,7 +75,17 @@ jobs:
             )
             SELECT image, package, vulnerability, version, type, severity FROM ruuuumble
```

Comment:
`ruuuumble` makes me laugh :)

Reply:
Hah, I thought I should put an Easter egg in there for future travelers, makes me smile every time too 😆